### PR TITLE
fix: Added Newsletter Body & Subject

### PR DESCRIPTION
**Pull request by noahonyejese**
**This PR:**
- This PR resolves minor issues and missing things detected by @sosiology and @KerstinFaye during the Internal Review (test) process.

**Fixes**
- Adds Subject and body to Newsletter
- Fixed English New Feature Subject Line "undefined"
- Showing the Accordion open on Datasets
- Overlapping Toggleswitch 

@bprusinowski there where still some minor mistakes that where causing issue's, this PR resolves all Testing Internal feedback from @sosiology

### CODE REVIEW

File: app/charts/shared/containers.tsx [hunk 1]
```diff
@@ -69,13 +69,14 @@ export const ChartSvg = ({ children }: { children: ReactNode }) => {
...
+        <foreignObject
           width={width - margins.right}
-          sx={{ display: "flex", textAlign: "right", height: 26 }}
+          y={20}
+          height="26"
+          style={{ display: "flex", textAlign: "right" }}
         >
           <CalculationToggle />
-        </Box>
+        </foreignObject>
```
Comment (bprusinowski) [inline app/charts/shared/containers.tsx:79]: @noahonyejese I am not sure if we need to revert this, I think the previous implementation missed `y: [18, 20]` in `sx`? 😄

Comment (noahonyejese) [reply app/charts/shared/containers.tsx:79]:
```javascript
No overload matches this call.
  Overload 1 of 2, '(props: { component: ElementType<any>; } & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<any> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>): Element', gave the following error.
    Type '{ display: "flex"; textAlign: "right"; y: number[]; height: number; }' is not assignable to type 'SxProps<Theme> | undefined'.
      Types of property 'y' are incompatible.
        Type 'number[]' is not assignable to type 'SystemStyleObject<Theme> | CssVariableType | ((theme: Theme) => string | number | SystemStyleObject<Theme>) | undefined'.
          Type 'number[]' is not assignable to type 'CSSSelectorObjectOrCssVariables<Theme>'.
            Index signature for type 'string' is missing in type 'number[]'.
  Overload 2 of 2, '(props: DefaultComponentProps<BoxTypeMap<{}, "div">>): Element', gave the following error.
    Type '{ display: "flex"; textAlign: "right"; y: number[]; height: number; }' is not assignable to type 'SxProps<Theme> | undefined'.
      Types of property 'y' are incompatible.
        Type 'number[]' is not assignable to type 'SystemStyleObject<Theme> | CssVariableType | ((theme: Theme) => string | number | SystemStyleObject<Theme>) | undefined'.ts(2769)
Box.d.ts(11, 7): The expected type comes from property 'sx' which is declared here on type 'IntrinsicAttributes & { component: ElementType<any>; } & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<...> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>'
Box.d.ts(11, 7): The expected type comes from property 'sx' which is declared here on type 'IntrinsicAttributes & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<any> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>'
(property) sx?: SxProps<Theme> | undefined
```

I can't add the array into sx

Comment (bprusinowski) [reply app/charts/shared/containers.tsx:79]: Yeah, looks like it's not supported for `y` :/

**Review by bprusinowski — APPROVED** [review]
LGTM 👌

**Pull request merged**